_includes/pbc/all/install-features/202212.0/install-the-dynamic-multiple-stores.md

```diff
@@ -0,0 +1,2544 @@
+            new DataImportConsole(DataImportConsole::DEFAULT_NAME . static::COMMAND_SEPARATOR . LocaleDataImportConfig::IMPORT_TYPE_LOCALE_STORE),
+            new DataImportConsole(DataImportConsole::DEFAULT_NAME . static::COMMAND_SEPARATOR . LocaleDataImportConfig::IMPORT_TYPE_DEFAULT_LOCALE_STORE),
+            new DataImportConsole(DataImportConsole::DEFAULT_NAME . static::COMMAND_SEPARATOR . StoreDataImportConfig::IMPORT_TYPE_STORE),            
+        ];
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Spryker\Shared\ApplicationExtension\Dependency\Plugin\ApplicationPluginInterface>
+     */
+    public function getApplicationPlugins(Container $container): array
+    {
+        $applicationPlugins = parent::getApplicationPlugins($container);
+
+        $applicationPlugins[] = new LocaleApplicationPlugin();
+
+        return $applicationPlugins;
+    }
+
+}
+```
+
+
+### Preparing csv files for configure stores, locales, currencies, countries via data import 
+
+Example for DE store locales configurations: 
+`data/import/common/DE/locale_store.csv`
+
+```
+locale_name,store_name
+en_US,DE
+de_DE,DE
+```
+
+Example for DE store default locale:
+`data/import/common/DE/default_locale_store.csv`
+
+```
+locale_name,store_name
+en_US,DE
+```
+
+Example for DE store currency-store configurations:
+`data/import/common/DE/currency_store.csv`
+
+```csv
+currency_code,store_name,is_default
+EUR,DE,1
+CHF,DE,0
+```
+
+Example for DE store coutry-store configurations:
+`data/import/common/DE/country_store.csv`
+
+```
+store_name,country
+DE,DE
+DE,FR
+```
+
+Use data import command for import configuration
+
+
+```bash 
+vendor/bin/console data:import:locale-store 
+vendor/bin/console data:import:default-locale-store
+vendor/bin/console data:import:currency-store   
+vendor/bin/console data:import:country-store
+```
+
+### Add translations
+
+Add new translation files for widget labls.
+
+**1. Append glossary according to your configuration:**
+
+By default, the glossary is located in the following directory:
+
+```
+data/import/common/common/glossary.csv
+```
+
+CSV file example:
+
+```csv
+store_widget.switcher.store,Store:,en_US
+store_widget.switcher.store,Shop:,de_DE
+```
+
+**2. Add the glossary keys use command:**
+
+```bash
+console data:import:glossary
+```
+
+**3. Generate new translation cache:**
+
+```bash
+console translator:generate-cache
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the configured data has been added to the spy_glossary table in the database.
+
+{% endinfo_block %}
+
+
+## Deploy file changes and enable dynamic store feature 
+
+Due to chnage ideology of region insted store configuration, you need to change deploy file for enable dynamic store feature.
+
+1. Change `deploy.yml` file for enable dynamic store feature.
```

Comment:
We don`t need to change existing files. I would say we can add a new one and use instead for DS setup